addon/components/simple-input.js

```diff
@@ -3,7 +3,7 @@ import humanize from "ember-form-builder/utilities/humanize";
 import guessType from "ember-form-builder/utilities/guess-type";
 
 const extension = {
-  i18n: Ember.inject.service(),
+  translationService: Ember.inject.service(),
```

Comment:
`formBuilderTranslations` suggests that there are **multiple** translations, when it's just an instance of the preferred translation service.  That sounds misleading.